### PR TITLE
Add flag selector to twilio channel

### PR DESCRIPTION
**Pull request by jugarrit**
@alavers 
@lemieux 
@dannytranlx 
@mspensieri 
@liyefei737

### CODE REVIEW

**Comment by jugarrit** [comment]
Added a `png` with the full list of flags.

Added a new vendored library, `react-telephone-input` directly into libs folder for the following reasons:

- Much easier to customize. The library had a lot of options but I wanted to add better `onInputValid` functionality and strip out code we weren't using to lessen our footprint.
- It made use of lots of lodash functions that are built in to ES6, these have been replaced
- etc.

Updated the SMS description.

Created new mixins to automatically set the width of a div based on the current size of the window.

This will allow us to have content correctly match the available width without having to copy/paste a bunch.

**Comment by lemieux** [comment]
:+1:

File: src/stylesheets/flags.less [hunk 1]
```diff
@@ -0,0 +1,796 @@
+@flag-width: 32px;
+@flag-height: 24px;
+
+// @flag-width: 80px;
+// @flag-height: 60px;
```
Comment (dannytranlx) [inline src/stylesheets/flags.less]: Comments?

**Comment by dannytranlx** [comment]
👍

File: src/stylesheets/flags.less (hunk 1 above)
Comment (jugarrit) [inline src/stylesheets/flags.less]: whupz

File: src/js/components/channels/twilio-channel-content.jsx [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
+            numberValid: false
+        });
+    }
 
-        }
-        return <span className='channel-content-value'>{ phoneNumber }</span>;
+    render() {
+        return <div>
+                   <ReactTelephoneInput ref='telInput'
+                                        defaultCountry='ca'
```
Comment (alavers) [inline src/js/components/channels/twilio-channel-content.jsx]: should be us perhaps?

**Comment by alavers** [comment]
👍

File: src/js/components/channels/twilio-channel-content.jsx (hunk 2 above)
Comment (dannytranlx) [inline src/js/components/channels/twilio-channel-content.jsx]: we talked about having it configurable in some previous PRs, don't know what was the outcome of that discussion

**Pull request merged**